Utilities/CreateTrustedCertificate.bat

```diff
@@ -45,8 +45,8 @@ IF %ERRORLEVEL% NEQ 0 GOTO :error
 ECHO.
 
 REM 1. Create strong certificate and private key, from https://stackoverflow.com/a/41366949/177710
-openssl.exe req -x509 -newkey rsa:4096 -sha256 -days 3650 -nodes -keyout "%CERTPATHBASE%.key" -out "%CERTPATHBASE%.crt" -subj "/CN=%DOMAIN%" -addext "subjectAltName=DNS:%DOMAIN%"
 IF %ERRORLEVEL% NEQ 0 GOTO :error
```

Comment:
How would this work, excactly?
![image](https://user-images.githubusercontent.com/1976647/111793756-db0e1480-88c5-11eb-8c66-f74b46d8c21b.png)